packages/types/src/Type.ts

```diff
@@ -88,6 +90,24 @@ export default class Type extends Text {
     };
   }
 
+  private _cleanupCompact (): Mapper {
```

Comment:
Not crazy about this - it needs to take `<T::Balance as HasCompact>::Type` (`::Type` and `T::` stripped next) and convert it to `Compact<T::Balance>`